braze/client.py

```diff
@@ -237,16 +290,16 @@ def create_braze_alias(self, emails, alias_label, attributes=None):
             self.track_user(attributes=attributes)
 
     def send_email(
-        self,
-        emails,
-        subject,
-        body,
-        from_email,
-        campaign_id=None,
-        recipient_subscription_state='all',
-        reply_to=None,
-        attachments=None,
-        override_frequency_capping=False
+            self,
+            emails,
+            subject,
+            body,
+            from_email,
+            campaign_id=None,
+            recipient_subscription_state='all',
+            reply_to=None,
+            attachments=None,
+            override_frequency_capping=False
```

Comment:
please revert these added tabs. this comment holds true everywhere.

Reply:
This is not proper indentation.

Reply:
I have reformatted the code to adhere to PEP 8 guidelines using the PyCharm reformatting tool.